Dynatrace/dynatrace-operator
# Add node collector token secret

## Description
https://dt-rnd.atlassian.net/browse/DAQ-1402

Token for communication between AG and nodeConfigCollector is reconciled and created. 

## How can this be tested?

Deploy the operator and a Dynakube with `kspm.enabled: true`, check if the token is created
Later check if the dynakube is changed to  `kspm.enabled: false`, that the secret is removed
